### PR TITLE
Add slora example for pipeline

### PR DESCRIPTION
**Pull request by AllentDan**

### CODE REVIEW

File: docs/en/inference/pipeline.md [hunk 1]
```diff
@@ -143,6 +143,27 @@ response = pipe(prompts, gen_config=gen_config)
...
+                                     adapters=dict(lora_name_1='chenchi/lora-chatglm2-6b-guodegang'))
+gen_config = GenerationConfig(top_p=0.8,
+                              top_k=40,
+                              temperature=0.8,
+                              max_new_tokens=1024)
+pipe = pipeline('THUDM/chatglm2-6b',
+                backend_config=backend_config)
+prompts = [[{
+    'role': 'user',
+    'content': '您猜怎么着'
```
Comment (grimoire) [inline docs/en/inference/pipeline.md:161]: I am not sure if it is a good idea to use Chinese example in English docs.

File: lmdeploy/serve/async_engine.py [hunk 2]
```diff
@@ -270,6 +270,7 @@ def __call__(self,
                  repetition_penalty: float = 1.0,
                  ignore_eos: bool = False,
                  do_preprocess: bool = True,
+                 adapter_name: Optional[str] = None,
```
Comment (lvhan028) [inline lmdeploy/serve/async_engine.py:273]: @irexyc will vl_pipeline be affected?

File: docs/en/inference/pipeline.md (hunk 1 above)
Comment (AllentDan) [reply docs/en/inference/pipeline.md:161]: Do you have any other examples?

**Review by grimoire — APPROVED** [review]

File: docs/en/inference/pipeline.md (hunk 1 above)
Comment (grimoire) [reply docs/en/inference/pipeline.md:161]: Nope

File: lmdeploy/serve/async_engine.py (hunk 2 above)
Comment (irexyc) [reply lmdeploy/serve/async_engine.py:273]: will not be affected

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**